cmd/mic/main.go

```diff
@@ -34,7 +34,7 @@ var (
 )
 
 func main() {
-	klog.InitFlags(nil)
+	// klog.InitFlags(nil)
```

Comment:
Please undo this change

Reply:
If we uncomment this, mic will fail at this line for some reason. Need to do more investigation on this.

Reply:
Synced with Anish and he suggests I can comment it for now. He will do more investigation.

Reply:
Synced with Anish and we should comment this for now or mic will fail. He will do more investigation on this. 

---
In reply to: [381815036](https://github.com/bowu24/aad-pod-identity/pull/1#discussion_r381815036) [](ancestors = 381815036)

Reply:
This means that all `klog` settings will have their default values. I do not think this is desirable

---
In reply to: [386088105](https://github.com/bowu24/aad-pod-identity/pull/1#discussion_r386088105) [](ancestors = 386088105,381815036)